src/styles/intent.styl

```diff
@@ -0,0 +1,52 @@
+@require 'settings/palette.styl'
+.app-wrapper
+    width 100%
+.app-content
+    height 100vh
+
+.intent-layout
+    height 100%
+    display flex
+    align-items stretch
+    flex-direction column
+
+
+.intent-footer
+    height 4.5rem
+    background-color paleGrey
+    box-shadow inset 0 -1px 0 0 silver
+    display flex
+    align-items center
+    padding .5rem 1rem
+
+    .intent-footer-label
+        flex 1
```

Comment:
I think `flex-grow 1` is more explicit than `flex 1` and it doesn't change the `flex-basis`, which is what you intend most of the time.

Reply:
:+1: but too late :/

Reply:
also, is the nesting really relevant? The naming should be enough to figure out its parent. No need for a nesting that add nothing but heavier selectors, I think.